.github/workflows/native-custom_marian-ubuntu.yml

```diff
@@ -0,0 +1,33 @@
+name: Native (Custom Marian) Ubuntu
```

Comment:
Remove file.

Reply:
We want to check the reduced Marian is working natively.  Otherwise we'll be in bigger bug territory later.